### ISSUE TITLE
Image support between Telegram and WhatsApp, GIFs not working, Images renamed to JFIF

### ISSUE THREAD

**Issue by null-von-sushi** [42wim/matterbridge#1292]
**Describe the bug**
Images sent from Whatsapp sometimes have issues.
Specifically, .gif images (sent from my friend via Gboard into WhatsApp) never arrive on my Bridged Telegram Group.
Photos get a .jfif file extension, which breaks a lot. Simply renaming them to .jpg allows them to be opened.

**To Reproduce**
 - Set up Telegram -> WhatsApp bridge.
 - 

**Expected behavior**
Images remain in the original format, or arrive as either standard PNG or JPG files.

**Screenshots/debug logs**
![1TG](https://user-images.githubusercontent.com/13616397/99312374-d8d37500-2855-11eb-9944-de1065565840.jpg)
![2WA](https://user-images.githubusercontent.com/13616397/99312375-d96c0b80-2855-11eb-9439-cd6da1551cfd.jpg)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Matterbridge version: 1.19.0 2bb21262

**Additional context**
I don't think it matters, but I use a WhatsApp Business account (technically, I just had to download the Business app because my phone numer isn't recognized as a phone number, but a landline). My friend uses a normal WhatsApp Account.

**Comment by Vetgans** [comment]
I don't use Telegram, but do use a Matrix <-> Whatsapp bridge.
Sending 'normal' jpg images works fine in both directions, however GIF's do cause problems:

GIF files are completely dropped, no mention whatsoever that something was sent.
Trying to fool Whatsapp and changing the extension from *.gif to *.mp4 doesn't fully work. Now at least it's mentioned that an *.mp4 was sent, but it doesn't play directly in Whatsapp. Real mp4's sent via the same bridge play fine.
You can download the renamed GIF-file though, and play it in a more sophisticated / dedicated player that automatically recognizes the real file format.

So i don't have problems with normal images, just the GIF's. Probably related to https://github.com/Rhymen/go-whatsapp/issues/154.

**Comment by null-von-sushi** [comment]
well renaming the weird "JFIF"  files to jpg seems to work without issues, so it probably would fix it if they were given the .jpg extension on the Telegram side of things?

**Comment by 42wim** [comment]
Added in #1316
Please test as I don't have whatsapp to test it myself

**Comment by null-von-sushi** [comment]
I compiled and set it up again, and it seems to work well from the testing I've done. GIF files sent from WhatsApp are renamed .m4v and seem to play after clicking on them, and normal images just displayed inline as expected.
Overall an improvement :)

One thing I noticed is that the session ID for WhatsApp doesn't seem to be reuseable for me, I have to log in every time, but I am not 100% sure why. I will investigate this further and open a relevant issue if necessary.

Thanks for the swift fix 👍
